include/saltatlas/common/data_reader.hpp

```diff
@@ -167,13 +177,21 @@ inline void read_points_with_id_helper(
     }
 
     std::string line_buf;
+    std::size_t line_num{0};
     while (std::getline(ifs, line_buf)) {
-      id_t       id{};
-      point_t    point;
-      const auto ret = parser(line_buf, id, point);
-      if (!ret) {
```

Comment:
I also want to check the values here (although I forgot to check point.empty() here..)